src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -331,14 +332,16 @@ uct_gdr_copy_rcache_mem_reg_cb(void *context, ucs_rcache_t *rcache,
                                uint16_t rcache_mem_reg_flags)
 {
     uct_gdr_copy_md_t *md = context;
-    int *flags = arg;
+    int *flags            = arg;
+    int silent            = (rcache_mem_reg_flags & UCS_RCACHE_MEM_REG_HIDE_ERRORS) ||
+                            (*flags & UCT_MD_MEM_FLAG_HIDE_ERRORS);
     uct_gdr_copy_rcache_region_t *region;
 
     region = ucs_derived_of(rregion, uct_gdr_copy_rcache_region_t);
     return uct_gdr_copy_mem_reg_internal(&md->super, (void*)region->super.super.start,
                                          region->super.super.end -
                                          region->super.super.start,
-                                         *flags, &region->memh);
+                                         *flags, silent, &region->memh);
```

Comment:
maybe, instead of adding new parameter, make sure that `UCT_MD_MEM_FLAG_HIDE_ERRORS` is passed in `flags` if `UCS_RCACHE_MEM_REG_HIDE_ERRORS` is set in `rcache_mem_reg_flags`